From a80e02a21e4b3faf74a1e807ae77b6496394dc0d Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Fri, 26 Jul 2013 10:06:56 +0900
Subject: [PATCH 1042/1083] Partially Revert "drm: Renesas R-Car Display Unit
 DRM driver"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This partially reverts commit dc035c5f16f24fa73bf669fdba685d05f1f18dc1
("drm: Renesas R-Car Display Unit DRM driver").
Specifically, it removes the shmobile/ from
drivers/gpu/drm/Makefile as that directory does not exist
and its presence in drivers/gpu/drm/Makefile causes
make clean to fail.

Reported-by: 片岡亮 <ryo.kataoka.wt@renesas.com>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit f5f2396fd19b28b501f5a1d5c5b81cc076ebe3be)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/gpu/drm/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/Makefile b/drivers/gpu/drm/Makefile
index 0119bc9..539d345 100644
--- a/drivers/gpu/drm/Makefile
+++ b/drivers/gpu/drm/Makefile
@@ -45,5 +45,4 @@ obj-$(CONFIG_DRM_EXYNOS) +=exynos/
 obj-$(CONFIG_DRM_GMA500) += gma500/
 obj-$(CONFIG_DRM_UDL) += udl/
 obj-$(CONFIG_DRM_RCAR_DU) += rcar-du/
-obj-$(CONFIG_DRM_SHMOBILE) +=shmobile/
 obj-y			+= i2c/
-- 
1.8.3.2

